src/video/H264Encoder.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "encoder.hpp"
+
+#include <string>
+#include <vector>
+
+#include <opencv2/core/core.hpp>
+
+namespace video {
+/**
+ * This class is a wrapper for the h264encoder library
+ */
+class H264Encoder {
+public:
+	/**
+	 * Uses the h264encoder library to encode webcam frames into encoded video data
+	 */
+	H264Encoder(int fps, int rf);
```

Comment:
Missing `@param` doxygen comments that explain what `fps` and `rf` mean. The constructor documentation should be sufficient for understanding it, users shouldn't have to go look at the comments on private fields.

Reply:
Also, this is where the min and max of `rf` should be explained. What do higher and lower numbers mean?